integrations/react-next-15/package.json

```diff
@@ -9,9 +9,9 @@
     "@js-temporal/polyfill": "^0.4.4",
     "@tanstack/react-query": "workspace:*",
     "@tanstack/react-query-devtools": "workspace:*",
-    "next": "^15.0.0-rc.0",
-    "react": "19.0.0-rc-4c2e457c7c-20240522",
-    "react-dom": "19.0.0-rc-4c2e457c7c-20240522",
+    "next": "^15.0.3",
+    "react": "19.0.0-rc-7ac8e612-20241113",
+    "react-dom": "19.0.0-rc-7ac8e612-20241113",
```

Comment:
can we please use `19.0.0-rc.1`, and use it _everywhere_. I’d really like to have the versions streamlined, thanks 🙏